config/routes.rb

```diff
@@ -2,4 +2,5 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :art_services
```

Comment:
do you think this makes more sense, or to only write the route you are creating here, namely #new?

like `resources :art_services, only: [:new, :create]`?